system/bluetooth_monitor/package.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>bluetooth_monitor</name>
+  <version>0.1.0</version>
+  <description>Bluetooth alive monitoring</description>
+  <maintainer email="fumihito.ito@tier4.jp">Fumihito Ito</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake_auto</buildtool_depend>
+
+  <depend>bluez</depend>
+  <depend>diagnostic_msgs</depend>
+  <depend>diagnostic_updater</depend>
+  <depend>fmt</depend>
+  <depend>libbluetooth-dev</depend>
+  <depend>rclcpp</depend>
+
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_lint_common</test_depend>
```

Comment:
```suggestion
  <test_depend>autoware_lint_common</test_depend>
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/862/commits/704c0075a0c2d4523a08df2eea78f2942d6eacb4